kyuubi-common/src/main/scala/org/apache/kyuubi/events/JsonEventLogger.scala

```diff
@@ -75,7 +75,13 @@ class JsonEventLogger[T <: KyuubiEvent](logName: String,
   }
 
   private def requireLogRootWritable(): Unit = {
-    val fileStatus = fs.getFileStatus(new Path(logRoot))
+    val fileStatus = try {
+      fs.getFileStatus(new Path(logRoot))
+    } catch {
+      case e: IOException =>
+        error(s"Get log directory $logRoot failed", e)
```

Comment:
what's the behavior before, do you mean we cannot log the exception ?

Reply:
Yes, we cannot log this exception.

Before fix this bug, when the dir is not exist, the engineLog only can get: java.lang.IllegalStateException: Illegal Service State: LATENT.
The cause of this exception is:
when we stop the engine before, we need to get this engine's engineStatus(SparkSQLEngine line 158).
this engineStatus will store the connectionUrl from frontendService. But if this serviceState is LATENT, it will throw this exception.